data/ui/ResolutionRow.ui

```diff
@@ -31,12 +30,12 @@
               </packing>
             </child>
             <child>
-              <object class="GtkLabel" id="active_label">
+              <object class="GtkLabel" id="status_label">
+                <property name="visible">True</property>
                 <property name="can_focus">False</property>
                 <property name="halign">end</property>
-                <property name="label" translatable="yes" comments="Translators: this is used to indicate the active resolution.">Active</property>
+                <property name="label" translatable="yes" comments="Translators: this is used to indicate the active status.">status</property>
```

Comment:
Will break translations. Probably not fixable since the label text changes dynamically. Any ideas?